source/_lovelace/area.markdown

```diff
@@ -13,12 +13,16 @@ The Area card lets you control and monitor an individual area.
 </p>
 
 
-To add the Area card to your user interface, click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Click the "Add Card" button in the bottom right corner and select **Area** from the card picker. All options for this card can be configured via the user interface.
+All options for this card can be configured via the user interface.
```

Comment:
Not sure I like these changes. For a user, they want an alarm card. They come here and learn exactly what to do

Reply:
Agree with @zsarnett.

If the YAML is documented we can list it is YAML,  but we should not be removing non-YAML instructions.

Reply:
I think we should follow the button card

https://www.home-assistant.io/lovelace/button/

Reply:
Yeah, that actually looks quite good. I will add back the descriptive text in question.

And I agree that all cards should have a description for the UI-options, but I don't like all options being repeated twice  (once for UI, once for YAML) like the button cards does.

Reply:
We don't want to overwhelm a user with YAML. So adding in Card Settings and basic description and then below adding YAML config is a better experience and easier for the eyes to read initially

Reply:
I agree that the "Card Settings" part looks nice. But what do you think about removing the description from the YAML-configuration-part and only having the types and required true/false there?

Reply:
Though there are some edge cases where settings are only available in YAML, I guess we could keep the descriptions in those cases.

Reply:
I believe this PR is an improvement over how it currently looks. Can we merge this as it is now? Or do you want me to make further changes? I don't think I will have time to make major changes in the near future.

Reply:
The text that this comment originally referred to is added again.